tests/measurements/test_measurements.py

```diff
@@ -125,33 +125,32 @@ class NoReturnTypeMeasurement(MeasurementProcess):
 
 
 def test_eq_correctness():
-    """Test that using `==` on two equivalent operators is True when both measurement
-    processes are the same object and False otherwise."""
+    """Test that using `==` on measurement processes behaves the same as
+    `qml.equal`."""
 
     class DummyMP(MeasurementProcess):
         """Dummy measurement process with no return type."""
 
-    mp1 = DummyMP(0)
-    mp2 = DummyMP(0)
+    mp1 = DummyMP(wires=qml.wires.Wires(0))
```

Comment:
did you have to make this wires change?

Reply:
Yeah, evidently `MeasurementProcess.__init__` expects the wires arg to already be a `Wires` object.